code/src/components/Artist.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+
+
+const Artist = (props) => {
+    return(
+        <div>
+            {props.item.artists.map((artist) => {
```

Comment:
It would be more readable if you use `props.item.artists.map((artist) => ( ) )` to avoid using return nested in another return